pkg/detectors/make/api_token/api_token_integration_test.go

```diff
@@ -0,0 +1,163 @@
+//go:build detectors
+// +build detectors
+
+package api_token
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+func TestMake_FromChunk(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*5)
+	defer cancel()
+	testSecrets, err := common.GetSecret(ctx, "trufflehog-testing", "detectors5")
+	if err != nil {
+		t.Fatalf("could not get test secrets from GCP: %s", err)
+	}
+	secret := testSecrets.MustGetField("MAKE_API_TOKEN")
+	inactiveSecret := testSecrets.MustGetField("MAKE_API_TOKEN_INACTIVE")
+
+	type args struct {
+		ctx    context.Context
+		data   []byte
+		verify bool
+	}
+	tests := []struct {
+		name                string
+		s                   Scanner
+		args                args
+		want                []detectors.Result
+		wantErr             bool
+		wantVerificationErr bool
+	}{
+		{
+			name: "found, verified",
+			s:    Scanner{},
+			args: args{
+				ctx:    context.Background(),
+				data:   []byte(fmt.Sprintf("You can find a make secret %s and endpoint us2.make.com", secret)),
```

Comment:
Could we also have this retrieve the endpoint from the secret manager instead of hard-coding it?

Reply:
@shahzadhaider1 Definitely! Can you put this endpoint in the secret manager as `MAKE_API_ENDPOINT` or was it named something else? I've updated the code locally but the tests don't pass currently.

Reply:
Gotcha, I'll define a `.env` file in the related directories for the tests then. Thanks!